<ui version="4.0" >
 <class>QDialogRoundText</class>
 <widget class="QDialog" name="QDialogRoundText" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text properties</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>3</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <layout class="QFormLayout" name="formLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="spinX" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>-61.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>61.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="spinY" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>-61.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>61.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QDoubleSpinBox" name="spinRadi" >
              <property name="minimum" >
               <double>10.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>80.000000000000000</double>
              </property>
              <property name="value" >
               <double>20.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Radius:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QDoubleSpinBox" name="spinAngle" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <double>360.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Angle:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3" >
            <property name="fieldGrowthPolicy" >
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QComboBox" name="comboAlignment" >
              <property name="currentIndex" >
               <number>0</number>
              </property>
              <item>
               <property name="text" >
                <string>Left</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Center</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Right</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>Alignment:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QComboBox" name="comboLocation" >
              <property name="currentIndex" >
               <number>0</number>
              </property>
              <item>
               <property name="text" >
                <string>Inside</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Outside</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>Location:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QFontComboBox" name="fontName" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="editable" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLabel" name="colorWidget" >
              <property name="minimumSize" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::Box</enum>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4" >
            <item row="0" column="0" >
             <widget class="QPushButton" name="btnFont" >
              <property name="text" >
               <string>Font...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="btnColor" >
              <property name="text" >
               <string>Color...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Text:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textEdit" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCDView" name="cdView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="interactive" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCDView</class>
   <extends>QGraphicsView</extends>
   <header>qcdview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnColor</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>spinX</tabstop>
  <tabstop>spinY</tabstop>
  <tabstop>spinRadi</tabstop>
  <tabstop>spinAngle</tabstop>
  <tabstop>comboAlignment</tabstop>
  <tabstop>comboLocation</tabstop>
  <tabstop>btnFont</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogRoundText</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogRoundText</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
